services/wallet/api.go

```diff
@@ -179,6 +179,13 @@ func (api *API) GetBalanceHistoryRange(ctx context.Context, chainIDs []uint64, a
 	return api.s.history.GetBalanceHistory(ctx, chainIDs, address, tokenSymbol, currencySymbol, fromTimestamp)
 }
 
+func (api *API) GetTokenList(ctx context.Context) ([]*token.TokenList, error) {
+	log.Debug("call to get token list")
+	rst := api.s.tokenManager.GetTokenList()
+	log.Debug("result from token list", "len", len(rst))
```

Comment:
Are those debug logs here intentionally?

Reply:
Yes we wrap all function around with debug log